pandas/tests/frame/methods/test_sort_index.py

```diff
@@ -761,6 +761,22 @@ def test_sort_index_with_categories(self, categories):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "ascending",
+        [
+            None,
+            (True, None),
+            (False, "True"),
+        ],
+    )
+    def test_sort_index_ascending_bad_value_raises(self, ascending):
```

Comment:
can you add the issue number as a comment

Reply:
Done.